typings/dojo/dojo.d.ts

```diff
@@ -255,6 +255,19 @@ declare module 'dojo/errors/create' {
 	export = create;
 }
 
+declare module 'dojo/errors/RequestError' {
+	interface RequestError extends Error {
+		name:string;
+		response:any;
+	}
+	var RequestError:{
+		new (name:string, response:any):RequestError;
+		name:string;
```

Comment:
Since this is defining the type of the variable it is defining a static 'name' and 'response' properties on RequestError which is probably not what you want.